src/bindings/html/dom.js

```diff
@@ -106,10 +112,21 @@ function camelCaseToDashed(string) {
 }
 
 function getElementTranslation(view, langs, elem) {
-  const l10n = getAttributes(elem);
+  const id = elem.getAttribute('data-l10n-id');
+
+  if (!id) {
+    return false;
+  }
+
+  const args = elem.getAttribute('data-l10n-args');
+
+  if (!args) {
+    return view.ctx.resolve(langs, id);
```

Comment:
This early return has the benefit of guarding against a `TypeError` if `data-l10n-args` is an empty string.